changelog_config.json

```diff
@@ -0,0 +1,62 @@
+{
+    "categories": [
+        {
+            "title": "## ❗ Deprecations",
+            "labels": [
+                "deprecation"
+            ]
+        },
+        {
+            "title": "## 🚀 New Features",
+            "labels": [
+                "enhancement"
+            ]
+        },
+        {
+            "title": "## 🐛 Bug Fixes",
+            "labels": [
+                "bug"
+            ]
+        },
+        {
+            "title": "## 📘 Documentation",
+            "labels": [
+                "documentation"
+            ]
+        },
+        {
+            "title": "## ⚙️ OmniSciDB",
+            "labels": [
+                "omniscidb"
+            ]
+        }
```

Comment:
The omniscidb label is applied to those RBC issues that solution ought to be implemented in omniscidb rather than in rbc. That said, there are omniscidb related PRs in rbc that may implement tests for omniscidb bug fix PRs or that pair with some omniscidb enhancement PR (for instance, when a omniscidb PR introduced the Column feature, there was a PR that implemented Column support in rbc).

However, I think instead of grouping rbc PRs with omniscidb label we should use enhancement or bug fix label for such PRs because it may be hard to decide whether omniscidb label should be applied or not (most changes in rbc are omniscidb related in the current development phase).

So, I suggest removing "OmniSciDB" section.

Reply:
Should we have "Testing" section?

Reply:
I'll replace the OmniSciDB tag by testing, then